UnityProject/Assets/Scripts/UI/UI Bottom/Hands.cs

```diff
@@ -122,6 +122,7 @@ public void Equip()
 		// Is there an item to equip?
 		if(!CurrentSlot.IsFull)
 		{
+			Logger.Log("!CurrentSlot.IsFull");
```

Comment:
also please: either remove logs after you test them or assign a category, in your case `Category.UI`